apiary-ranger-metastore-plugin/src/main/java/com/expediagroup/apiary/extensions/rangerauth/policyproviders/ApiaryRangerAuthAllAccessPolicyProvider.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.rangerauth.policyproviders;
+
+import com.expediagroup.apiary.extensions.rangerauth.listener.ApiaryRangerAuthPreEventListener;
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.ranger.admin.client.RangerAdminClient;
+import org.apache.ranger.admin.client.RangerAdminRESTClient;
+import org.apache.ranger.plugin.model.RangerPolicy;
+import org.apache.ranger.plugin.util.GrantRevokeRequest;
+import org.apache.ranger.plugin.util.ServicePolicies;
+import org.apache.ranger.plugin.util.ServiceTags;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class to return a list of Ranger authorization policies that permits all access.  If this is configured in
+ * ranger-hive-security.xml in the "ranger.plugin.hive.policy.source.impl" property, Ranger will allow all actions.
+ * <p>
+ * The intent of this class is to only configure it on the Apiary read-only metastore so that Ranger auditing can still
+ * be used without Ranger authorization policies.  We have to do it this way, because Ranger auditing is down in the
+ * guts of the policy evaluation code, so there isn't a cleaner way to audit without policies that I can find.
+ */
```

Comment:
Close the `</p>`

Reply:
I didn't even see that there was a `<p>`.  I think IntelliJ helpfully did that for me...

Reply:
Interesting, Oracle specifically says to use just a `<p>` tag - https://www.oracle.com/technetwork/articles/java/index-137868.html.  But IntelliJ definitely needs the closing tag to render the blank link in its Javadoc viewer.

Reply:
Yeah, I know, we've had many discussions about this in other projects. Proper (x)HTML would obviously close it and some rendering engines of Javadoc prefer/require it while others seem to stick to Oracle's definition of not having the closing tag.